components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2458,7 +2472,10 @@ void RewardsServiceImpl::SetContributionAutoInclude(
       ? ledger::PUBLISHER_EXCLUDE::EXCLUDED
       : ledger::PUBLISHER_EXCLUDE::INCLUDED;
 
-  bat_ledger_->SetPublisherExclude(publisher_key, status);
+  auto callback = base::BindOnce(
+    &RewardsServiceImpl::OnSetContributionAutoInclude,
```

Comment:
What if you just bind to `OnExcludedSitesChanged`, based on the fact that you just duplicated it. This way you don't need another function?

Reply:
fair point, will remove

Reply:
updated so `OnSetContributionAutoInclude` calls `OnExcludedSitesChanged` to avoid having to refactor `OnSetContributionAutoInclude` to use `int32_t` instead of `enum` type